test/integration/README.rst

```diff
@@ -5,7 +5,8 @@ aws-encryption-sdk Integration Tests
 In order to run these integration tests successfully, these things must be configured.
 
 #. Ensure that AWS credentials are available in one of the `automatically discoverable credential locations`_.
-#. Set environment variable ``AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID`` to valid
+#. Set environment variable ``AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID``
+   and ``AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID_2`` to valid
```

Comment:
Do we have documentation on `AWS_KMS_KEY_ID` vs `AWS_KMS_KEY_ID_2`? Are there times when we would want to set one to a value and 2 to a different value?

Edit: `integration_test_utils.py` makes this clearer. Would be good to also note why we have both here as well, though.